### PR TITLE
2.0.0beta6 release notes

### PR DESCRIPTION
**Pull request by pmaier1**
Let me know or add yourself if something important is missing. THX.

### CODE REVIEW

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 1]
```diff
@@ -23,9 +23,33 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
 - https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 6
+
+Infinite Scale 2.0.0 beta6 includes a huge number of bug fixes, cosmetic and performance improvements. It also brings a large number of bug fixes further hardening the 'Spaces' feature.
```
Comment (micbar) [inline modules/ROOT/pages/ocis_release_notes.adoc:28]:
```suggestion
Infinite Scale 2.0.0 beta6 includes a huge number of bug fixes, cosmetic and performance improvements. It also brings a lot of improvements and fixes which further hardened the 'Spaces' feature.
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 2]
```diff
@@ -23,9 +23,33 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
...
+
+* The drop area for drag & drop uploads is now shown again. https://github.com/owncloud/web/issues/7080[web#7080]
+* Deleting multiple files at once now works properly. https://github.com/owncloud/web/pull/7357[web#7357]
+* Some bugs on the 'Shares' page have been addressed (share indicators, declining shares). https://github.com/owncloud/web/pull/7379[web#7379] https://github.com/owncloud/web/pull/7355[web#7355]
+* The file list rendering performance has been improved. https://github.com/owncloud/web/issues/7038[web#7038]
+* Upload stability in Web has been improved with a better access token handling. https://github.com/owncloud/web/issues/7240[web#7240]
+* The file versions of a shared file will now be shown. https://github.com/owncloud/web/pull/7313[web#7313]
+* Contextual helper texts for users have been improved. https://github.com/owncloud/web/pull/7404[web#7404]
+* The drop menus in the right sidebar (e.g., role picker) have been visually aligned and improved. https://github.com/owncloud/web/pull/7365[web#7365]
+* File search will now report the number of results shown (if there are more than actually shown). https://github.com/owncloud/web/pull/7267[web#7267]
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:40]:
```suggestion
* File search will now report the total number of results (if there are more than actually shown). https://github.com/owncloud/web/pull/7267[web#7267]
```

**Review by micbar — APPROVED** [review]
Small nitpick

**Comment by mmattel** [comment]
I am going to publish the helm charts documentation tomorrow. @wkloucek @micbar the hc are usable for beta.4/5. what about beta.6? There is only one small change necessary in docs to enable it, to be checked in the charts repo. Saying that, we could announce hc in the release notes if wanted, the link already exists, only the content gets an update.

**Pull request merged**